internal/emulator/start.go

```diff
@@ -77,23 +77,22 @@ func ConfiguredServiceKey(
 		}
 	}
 
-	serviceAccount, _ := proj.EmulatorServiceAccount()
+	serviceAccount, err := proj.EmulatorServiceAccount()
+	if err != nil {
+		util.Exit(1, err.Error())
+	}
 
-	serviceKeyHex, ok := serviceAccount.DefaultKey().(*project.HexAccountKey)
-	if !ok {
+	privateKey, err := serviceAccount.Key().PrivateKey()
+	if err != nil {
 		util.Exit(1, "Only hexadecimal keys can be used as the emulator service account key.")
```

Comment:
Does this error message still apply here?

Reply:
Yes it does, as you potentially could provide an account with the KMS key, and that private key can not be fetched.